mock/py/mockbuild/util.py

```diff
@@ -775,6 +783,9 @@ def _prepare_nspawn_command(chrootPath, user, cmd, nspawn_args=None, env=None,
     if _check_nspawn_resolv_conf():
         nspawn_argv.append("--resolv-conf=off")
 
+    if check_nspawn_has_suppress_sync_option():
+        nspawn_argv += ['--suppress-sync=yes']
```

Comment:
Is nosync something to be set by default? Even Fedora Koji is using nspawn nowadays, so the change affects mostly every build out there (not konflux, no nspawn there).